### PR TITLE
Wave2 852 contact info

### PR DESCRIPTION
**Pull request by nindusaini**

### CODE REVIEW

File: sites/all/modules/ucalgary/ucgy_fgs/templates/node--fgs-graduate-degree.tpl.php [hunk 1]
```diff
@@ -89,9 +89,51 @@
...
   <h2><?php echo t('Related programs')?></h2>
   <p><?php echo t('If you\'re interested in this program you might want to explore other programs that UCalgary offers.')?></p>
+</div>
 
+<!--Contact Area-->
 
+<h2><?php echo t('Go to the faculty and department pages for more information');?></h2>
+<!--Display the complete address from faculty entity reference field-->
+<div>
+    <!--There can be more than one faculty references hence the foreach look-->
```
Comment (robbdavis) [inline sites/all/modules/ucalgary/ucgy_fgs/templates/node--fgs-graduate-degree.tpl.php]: foreach loop ;)

File: sites/all/modules/ucalgary/ucgy_fgs/templates/node--fgs-graduate-degree.tpl.php [hunk 2]
```diff
@@ -89,9 +89,51 @@
...
+        </h3>
+        <p>
+            <!--Display the required address fields-->
+            <?php echo $faculty['entity']->field_faculty_address[LANGUAGE_NONE][0]['thoroughfare'] ;?><br>
+            <?php echo $faculty['entity']->field_faculty_address[LANGUAGE_NONE][0]['premise'] ;?><br>
+            <?php echo $faculty['entity']->field_faculty_address[LANGUAGE_NONE][0]['locality']
+                .', '. $faculty['entity']->field_faculty_address[LANGUAGE_NONE][0]['administrative_area'] . ' '.
+                $faculty['entity']->field_faculty_address[LANGUAGE_NONE][0]['postal_code'];?><br>
+            <!--Display the faculty phone number-->
+            <?php echo $faculty['entity']->field_faculty_phone[LANGUAGE_NONE][0]['value'];?>
```
Comment (robbdavis) [inline sites/all/modules/ucalgary/ucgy_fgs/templates/node--fgs-graduate-degree.tpl.php]: Are the above all required fields? If no, do we need if isset() statements before printing?

File: sites/all/modules/ucalgary/ucgy_fgs/templates/node--fgs-graduate-degree.tpl.php [hunk 3]
```diff
@@ -89,9 +89,51 @@
...
+    </h3>
+    <p>
+        <!--Display the required address fields-->
+        <?php echo $content['field_fgs_grad_degree_department']['#items'][0]['entity']->field_department_address[LANGUAGE_NONE][0]['thoroughfare'] ;?><br>
+        <?php echo $content['field_fgs_grad_degree_department']['#items'][0]['entity']->field_department_address[LANGUAGE_NONE][0]['premise'] ;?><br>
+        <?php echo $content['field_fgs_grad_degree_department']['#items'][0]['entity']->field_department_address[LANGUAGE_NONE][0]['locality']
+        .', '. $content['field_fgs_grad_degree_department']['#items'][0]['entity']->field_department_address[LANGUAGE_NONE][0]['administrative_area'] . ' '.
+            $content['field_fgs_grad_degree_department']['#items'][0]['entity']->field_department_address[LANGUAGE_NONE][0]['postal_code'];?><br>
+        <!--Display the department phone number-->
+        <?php echo $content['field_fgs_grad_degree_department']['#items'][0]['entity']->field_department_phone[LANGUAGE_NONE][0]['value'];?>
```
Comment (robbdavis) [inline sites/all/modules/ucalgary/ucgy_fgs/templates/node--fgs-graduate-degree.tpl.php]: Same comment as above. Do we need to check if these are set before printing?

**Comment by robbdavis** [comment]
Hi set to approved and then I saw a minor issue on probo. Comments in jira ticket.

**Comment by nindusaini** [comment]
Created a new PR because of conflicts and closing this one

**Pull request closed without merging**